packages/headless/src/components/popover/popover.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+  component$,
+  createContext,
+  Slot,
+  useClientEffect$, useContextProvider,
+  useSignal,
+  useStore,
+} from "@builder.io/qwik";
+import { autoUpdate, computePosition, flip, shift, } from "@floating-ui/dom";
+
+// context
+interface ContextProps { isOpen: boolean }
+export const myContext = createContext<ContextProps>('my-context');
+
+// popover component
+interface PopoverProps {
+  alignments?: 'auto' | 'left' | 'top' | '...';
+}
+
+export const Popover = component$((props: PopoverProps) => {
+  const state = useStore({ isOpen: false });
```

Comment:
Please replace with useSignal

Reply:
are you sure @gilf ? 
It seems I cannot update the `isOpen` context property from a children component anymore (it lose the reactivity).
Or I wrong? In fact I see in the documentation it uses `useStore` to read and writea simple `counter`